src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -93,12 +93,14 @@ public async Task<IActionResult> Login(string returnUrl = null)
                     var provider = schemes.First().Name;
 
                     var dataProtector = _dataProtectionProvider.CreateProtector(nameof(DefaultExternalLogin))
-                                            .ToTimeLimitedDataProtector();
+                       .ToTimeLimitedDataProtector();
 
                     var token = Guid.NewGuid();
                     var expiration = new TimeSpan(0, 0, 5);
                     var protectedToken = dataProtector.Protect(token.ToString(), _clock.UtcNow.Add(expiration));
-                    await _distributedCache.SetAsync(token.ToString(), token.ToByteArray(), new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = expiration });
+                    await _distributedCache.SetAsync(token.ToString(),
+                        token.ToByteArray(),
+                        new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = expiration });
                     return RedirectToAction(nameof(DefaultExternalLogin), new { protectedToken, returnUrl });
```

Comment:
Suggested change

                    var protectedToken = dataProtector.Protect(token.ToString(), _clock.UtcNow.Add(expiration));

                    await _distributedCache.SetAsync(
                        token.ToString(),
                        token.ToByteArray(),
                        new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = expiration });

                    return RedirectToAction(nameof(DefaultExternalLogin), new { protectedToken, returnUrl });

Reply:
> 
> 
> Suggested change
> 
> ```
>                 var protectedToken = dataProtector.Protect(token.ToString(), _clock.UtcNow.Add(expiration));
> 
>                 await _distributedCache.SetAsync(
>                     token.ToString(),
>                     token.ToByteArray(),
>                     new DistributedCacheEntryOptions() { AbsoluteExpirationRelativeToNow = expiration });
> 
>                 return RedirectToAction(nameof(DefaultExternalLogin), new { protectedToken, returnUrl });
> ```

I cant' see the suggestion actually